src/scenes/home/branding/logo/logo.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+import styles from './logo.css';
+
+class BrandingLogo extends Component {
+  render() {
+    return (
+      <Section title="Logo">
+        <p className={styles.logosInfo}>
+          The size ratio between the star and the medallion changes depending
+          on the size of reproduction. Please make use of the appropriate
+          sized logo when creating collateral.
+        </p>
+        <p className={styles.logosInfo}>
+          In most cases, use the blue-accent version of the logo.
+          The red-accent is delivered for special uses only.
+        </p>
```

Comment:
Basically,

![screen shot 2018-01-20 at 9 11 08 pm](https://user-images.githubusercontent.com/9523719/35191069-7889e992-fe26-11e7-95e3-5df6d1bf70eb.png)

is missing after this point in code.

Reply:
@reddysridhar53 see ^